src/client.ts

```diff
@@ -30,6 +30,9 @@ const gotRetryOptions: Partial<RequiredRetryOptions> = {
   methods: ['POST'],
 };
 
+// small page size if test so we can validate pagination
+const PAGE_SIZE = process.env.LOAD_ENV ? 2 : 100;
```

Comment:
Can we mock up the request or expose something else. I realize this is low risk but generally like to avoid changing code to fit tests.

Actually, can you record tests with the value set to 2. Then switch it back to 100. That way the recordings are already set?

Reply:
already recorded with value set to 2. I have now removed, and tests are passing with the page size of 2 👍